CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # Changelog
 
+## v6.0.0
+
+Warning: There are Breaking Changes
+
+### New features
+- New module [IdentitySdkWeChat](IdentitySdkWeChat) for native WeChat Login Integration 
+- Add optional `origin` parameters to most authentification functions to help categorize your traffic
+
+### Bug fix
+- Fix passwordless flow with email
+
+### Breaking changes
+ - The functionality of module `IdentitySdkWebView` has been integrated into the main `IdentitySdkCore` module. Therefore `IdentitySdkWebView` has been deprecated.
```

Comment:
```suggestion
- The functionality of module `IdentitySdkWebView` has been integrated into the main `IdentitySdkCore` module. Therefore `IdentitySdkWebView` has been deprecated.
```

Reply:
> Therefore `IdentitySdkWebView` has been deprecated.

Just deprecated? Or removed?

Reply:
It will still be available in versions up to 5.8.0